frontend/constants/prompts.js

```diff
@@ -0,0 +1,25 @@
+const categorizePrompts = [
```

Comment:
Let's create a collection called "personas" and create the db schema for this collection and fetch them when the user loads the dicovery library.